rialto-ocdm.supp

```diff
@@ -0,0 +1,73 @@
+#
+# If not stated otherwise in this file or this component's LICENSE file the
+# following copyright and licenses apply:
+#
+# Copyright 2023 Sky UK
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Suppression logic for valgrind
+
+{
+   ignore_libgobject-2.0_register_fundamental
+   Memcheck:Leak
+   ...
+   obj:*/libgobject-2.0.so*
+   fun:g_type_register_fundamental
+}
+{
+   ignore_libgobject-2.0_specific_leak
+   Memcheck:Leak
+   match-leak-kinds: possible
+   fun:calloc
+   fun:g_malloc0
+   obj:*/libgobject-2.0.so*
+   obj:*/libgobject-2.0.so*
+   ...
+   fun:call_init
+   fun:_dl_init
+   obj:*/ld-*
+}
+{
+   ignore_glib_type_leaks
+   Memcheck:Leak
+   fun:*alloc
+   ...
+   fun:g_type_register_static
+}
+{
+   ignore_gtype_class_ref_leak
+   Memcheck:Leak
+   ...
+   fun:g_type_class_ref
+}
+{
+   ignore_gst_init_leak
```

Comment:
Can you try calling gst_deinit at the end of test/suite? Maybe this suppresion will not be needed.

Reply:
I checked it now, gst_deinit doesn't help - suppression is still needed.